duckduckgo/iOS
# clean up test targets and schemes

Task/Issue URL: https://app.asana.com/0/414709148257752/1199564566179646
Tech Design URL:
CC:

**Description**:

* Unit Tests are unchanged
* Integration tests now run tests that do not require a UI test, but make some kind of external conneciton
* Fingerprinting UI tests - cleaned up the scheme to just run the fingerprinting ui tests
* Atb UI Tests - runs the ATB tests that involve the UI
* DuckDuckGo scheme now runs "Unit Tests" and "Integration Tests" so will get executed frequently (they are fast)

Bitrise:

* Integration test scheduled job updated to run Atb UI Test scheme


**Steps to test this PR**:
1. Check out the code
1. With clean simulator run tests for each scheme (reset simulators between each UI tests scheme)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
